src/styles/miscStyles.js

```diff
@@ -5,6 +5,9 @@ export const statics = {
   largerText: {
     fontSize: 20,
   },
+  boldText: {
+    fontWeight: 'bold',
+  },
```

Comment:
```
account-info: Set User Name to bold

fixes request on #5515 for bold user name and same size email text
```

It sounds like this commit is meant to "fix" a style that was introduced in the previous commit. Instead of adding a new commit, our practice is to simplify the commit history by not introducing discarded ideas at all. In this case, we can do that by [squashing](https://zulip.readthedocs.io/en/latest/git/fixing-commits.html#squashing-commits) this commit with the previous one.

Reply:
Also, rather than adding to `miscStyles`, I'd prefer to write the `fontWeight: 'bold'` style locally in the file where it's used. That way, less work is needed (e.g., ⌘-clicking through in VSCode) for a reader to understand what style is actually getting applied. That's true of this bit of UI, and would also be true of other bits of UI that want to apply this style. The `miscStyles` file was added in an earlier era of the mobile-app project, and we've lately been trimming it down incrementally; if you're interested, you can [view the history of changes to the file](https://github.com/zulip/zulip-mobile/blob/main/docs/howto/git.md#git-log-secret).